Pipfile

```diff
@@ -8,11 +8,11 @@ apispec = {extras = ["yaml"], version = "==6.3"}
 apispec-webframeworks = "==0.5.2"
 cookiecutter = "==2.1.1"
 deepdiff = "==6.7.1"
-flask = "==3.0.0"
+flask = "==3.1.0"
 flask-cors = "==5.0.0"
-flask-socketio = "==5.3.6"
+flask-socketio = "==5.4.1"
 Flask-RESTful = ">=0.3.9"
-gevent = "==23.7.0"
+gevent = "==24.11.1"
```

Comment:
why don't we specify a version for pyarrow ? especially as we want it between 16 & 18 ?

Reply:
I agree Pyarrow version should be explicitly set. I would set it to 16.

Reply:
i will add it